tests/api/test_siblings.py

```diff
@@ -439,9 +443,329 @@ def test_node_siblings(client, resource, uid, backward_rel, expected):
+                'name': 'some_build',
+                'original_nvr': None,
+                'owner': None,
+                'release': '4.el7_4_as',
+                'resource_type': 'ContainerKojiBuild',
+                'start_time': '2018-04-02T19:39:06+00:00',
+                'state': 2,
+                'tags': [],
+                'triggered_by_freshmaker_event': {
+                    'event_type_id': 8,
+                    'id': '1180',
+                    'message_id': 'ID:messaging-devops-broker01.test',
+                    'state': 2,
+                    'state_name': 'COMPLETE',
+                    'state_reason': 'All container images have been rebuilt.'
+                },
+                'version': '1.7.5'
+            },
+            {
+                'advisories': [{
+                    'actual_ship_date': '2017-08-01T15:43:51+00:00',
+                    'advisory_name': 'RHBA-2017:2251-03',
+                    'content_types': ['docker'],
+                    'created_at': '2017-04-03T14:47:23+00:00',
+                    'id': '12327',
+                    'issue_date': '2017-08-01T05:59:34+00:00',
+                    'product_name': 'Red Hat Enterprise Linux',
+                    'product_short_name': 'RHEL',
+                    'release_date': None,
+                    'security_impact': 'None',
+                    'security_sla': None,
+                    'state': 'SHIPPED_LIVE',
+                    'status_time': '2017-08-01T15:43:51+00:00',
+                    'synopsis': 'cifs-utils bug fix update',
+                    'update_date': '2017-08-01T07:16:00+00:00'
+                }],
+                'commit': None,
+                'completion_time': '2017-04-02T19:39:06+00:00',
+                'creation_time': '2017-04-02T19:39:06+00:00',
+                'epoch': '0',
+                'extra': None,
+                'id': '710',
+                'module_builds': [],
+                'name': 'slf4j_2',
+                'original_nvr': None,
+                'owner': None,
+                'release': '4.el7_4_as',
+                'resource_type': 'ContainerKojiBuild',
+                'display_name': 'slf4j_2-1.7.4-4.el7_4_as',
+                'start_time': '2017-04-02T19:39:06+00:00',
+                'state': 1,
+                'tags': [],
+                'triggered_by_freshmaker_event': {
+                    'event_type_id': 8,
+                    'id': '1180',
+                    'message_id': 'ID:messaging-devops-broker01.test',
+                    'state': 2,
+                    'state_name': 'COMPLETE',
+                    'state_reason': 'All container images have been rebuilt.'
+                },
+                'version': '1.7.4'
+            }
+        ],
+        'meta': {
+            'description': 'Container builds attached to RHBA-2017:2251-03'
+        }
+    })
+])
+def test_module_story_node_siblings(client, resource, uid, backward_rel, story_path, expected):
```

Comment:
You should remove `story_path` since it's always just set to `module` since this test is only for module siblings.